playbooks/enable_bulk_emails/enable_bulk_emails.yml

```diff
@@ -0,0 +1,10 @@
+---
+# A simple one-off playbook that allows starting and stopping instance services.
+# This is used by Ocim to stop celery workers from inactive appservers to pick
+# up tasks.
```

Comment:
@mtyaka, looks like this comment was copied from other playbook.

Reply:
You're right :/ I'll open a PR to fix this in the upcoming sprint.

Reply:
I added a commit to update this text to a PR I'm currently working on: https://github.com/open-craft/opencraft/pull/782